common/lib/dependabot/errors.rb

```diff
@@ -191,6 +191,11 @@ def self.updater_error_details(error)
         "error-type": "private_source_authentication_failure",
         "error-detail": { source: error.source }
       }
+    when Dependabot::MissingDependencyInRegistry
```

Comment:
```suggestion
    when Dependabot::DependencyNotFound
```

Reply:
The package is not missing from the registry, the package is missing from the lockfile, so suggesting better name.